### ISSUE TITLE
Configure available storage classes

### ISSUE THREAD

**Issue by kywalker-rh** [opendatahub-io/odh-dashboard#1798]
Related Jira issue: https://issues.redhat.com/browse/RHODS-6909

**Comment by andrewballantyne** [comment]
@kywalker-rh Notebook (NB) Sizes (#1832) & Storage Classes (this ticket) are two different things. I do believe this is still needed.

We have mocks in https://github.com/opendatahub-io/odh-dashboard/issues/1270 -- so maybe this ticket isn't needed all the same?

**Comment by kywalker-rh** [comment]
Good point @andrewballantyne, this should say its covered by #1270 instead of #1832. I updated the comment closing the issue.
